Makefile

```diff
@@ -107,7 +107,10 @@ slow-odgi-all-oracles: og
 	-turnt --save --env overlap_oracle test/*.og
 	-turnt --save --env paths_oracle test/*.og
 
-slow-odgi-all-tests: slow-odgi-all-oracles
+# In reality slow-odgi-all-tests needs slow-odgi-all-oracles as a dependency.
+# Run the below by itself ONLY if you know that the GFAs have not changed,
+# in which case slow-odgi-all-oracles would have had no effect anyway.
+slow-odgi-all-tests:
```

Comment:
Here's an alternate way of getting the quieter testing system I was after with the unzipping strategy.

`test-slow-odgi` now builds all the oracles and all the tests (here I am assuming that when you write 
`big_target: small_target_1 small_target2`, then `small_target_1` is run before `2`).

Then `slow-odgi-all-tests` has _no_ dependency as far as the Makefile knows; it is up to the user to use this quick version when they know for sure that the GFAs have not changed.